plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTable.java

```diff
@@ -61,8 +64,9 @@ public TestCassandraTable(
                 sqlExecutor.execute(format("INSERT INTO %s.%s (%s) VALUES (%s)", keyspace, tableName, columns, row));
             }
 
-            // Ensure that the currently created table is visible to other sessions (e.g. the session used in DistributedQueryRunner)
-            server.refreshSizeEstimates(keyspace, tableName);
+            // Ensure that the currently created table is visible to Trino
```

Comment:
was this flaky because of the default of `cassandra.consistency-level` being `ONE` instead of `QUORUM`?

Reply:
The flaky issue still remains with QUORUM as far as I confirmed locally.